server/scripts/round.js

```diff
@@ -41,19 +41,19 @@ module.exports = class Round {
         this.io.in(this.lobbyName).emit('current-jackpot', this.jackpot)
         switch (percentDone) {
           case 33:
-            this.io.in(this.lobbyName).emit('announce-jackpot', {
+            this.io.in(this.lobbyName).emit('update', {
```

Comment:
What if you had a pure function that, given a `percentDone`, could return you the string "X time left!"? That would clean this up quite a bit.